content/posts/2019-08-03---Going-from-ember-to-vue.md

```diff
@@ -146,14 +156,17 @@ I just wanted some examples of the major key pieces.
 First off, all of the markup is wrapped in a `template`.
 After that, you are writing normal HTML.
 You call other components just like `<AngleBracket />` components in Ember, but the differences (for now) is that you have to import them.
-This saves from potential name collision, as an npm package might have a component with the same name as one of yours.
-You could do `import SomeCustomComponent as Monkey from '@something/components';` and rename a component without any magic.
+This saves you from potential name collision, as an npm package might have a component with the same name yours.
+You could do `import SomeCustomComponent as Monkey from '@something/components';` and rename a component easily.
+Again, this is something in an RFC for Ember already but it cements a point I made in an earlier post that [the resolver is a bit too clever sometimes](/posts/a-few-thoughts-on-ember/#3-make-the-resolver-less-clever).
```

Comment:
:) this is a WIP!

it's actually a primary motivator behind why module unification hadn't/hasn't happened -- too much resolver logic.